web/client/src/styles/request.css

```diff
@@ -0,0 +1,3 @@
+.request-content {
+  margin-top: 2rem !important;
```

Comment:
Remove `!important`

Reply:
important needs to be here or there will be no spacing between the parts